### DISCUSSION TITLE
Looooopy's build - Done!

### DISCUSSION THREAD

**Discussion by Looooopy** [joshreve/dactyl-keyboard#68] [Show and tell]
![image](https://user-images.githubusercontent.com/16755871/145065145-54c947db-090a-41d6-b60f-105f5536f185.png)

**Comment by Looooopy** [comment]
16h print time

![image](https://user-images.githubusercontent.com/16755871/145175901-09a9c513-e89d-4dd5-9976-0a0b3f2b0b06.png)

**Reply by joshreve** [reply]
Man I wish I had the patience to print at that resolution.  Looking forward to seeing it when it is done.  Is that at .1 layer thickness?

**Reply by Looooopy** [reply]
its 0.2 layer heights.

And here is my bad output - I think my retraction was to high so when it got more detailed I got screwed :) 
I will have to sand it down and put some spray putty and then sand one time more and put white spray paint on it.
See if I get my print right on the right side

![image](https://user-images.githubusercontent.com/16755871/145617784-16c71f83-d6b8-4c25-a730-116cd1a89500.png)

![image](https://user-images.githubusercontent.com/16755871/145617906-cfb405c3-c17a-4e54-a6dc-835a62ad2cad.png)

**Comment by Looooopy** [comment]
It was my PTFE tube that got jerked up a small portion after 20h of printing and made a partial clog.

So the next time I do two changes 
1. Print me a Bowden Extruder Clip
2. To get less imperfection that I need to mange after lower my retraction distance a small portion (to 5mm from 7mm).

Wish me luck *smile*!

Bad that I need to put putty the whole model to make it smooth this time!

**Reply by joshreve** [reply]
Yeah, the bowden tube quick connects are a bit shoddy.  I also had to totally dismantle and clear out the head due to gapping of the tube and subsequent partial clogs like you described.  I've seriously considered getting some proper tubing ferrules, but the extruder isn't designed to accept them.  I used to develop chromatography instruments and there are some 1/4-20 ferrule fittings that would be perfect if I ever decided to custom machine an extruder head.  Until then, all you can do is keep an eye on it.  Come to think of it, I may mark my tube just above the connection so I can see if it started slipping.  

At least it finished and can be cleaned up.  Inconvenient, but not undoable.

**Reply by Looooopy** [reply]
I think I found a solution for the bad connection, I will soon try out: https://www.youtube.com/watch?v=dIkjR2Ytx-g 

Here I have done a couple of coats with spray putty and it looks a lot better, tonight I will paint it white again.

![image](https://user-images.githubusercontent.com/16755871/146041215-0095ce7b-c829-44b6-b139-6c0b0169c143.png)
![image](https://user-images.githubusercontent.com/16755871/146041337-45f246d6-5e74-4ed4-b02b-0f054eddb910.png)


This emblem I will airbrush just below the OLED with a base of black I suppose but I will try it out on paper to get the hang of how it will look, do you know what it is :)

![image](https://user-images.githubusercontent.com/16755871/146043444-4f0602a1-b39e-4c75-a104-2eafbb054e4f.png)

**Reply by Looooopy** [reply]
Now it got it first coat of white see if I need to do a second one tomorrow.

![image](https://user-images.githubusercontent.com/16755871/146095876-20d22bdb-973e-41c7-8987-ead38e9181e6.png)
![image](https://user-images.githubusercontent.com/16755871/146095929-c3a55475-4d20-4405-b75a-6239f4c88738.png)

**Reply by joshreve** [reply]
Looking good.  I like the predator insignia.  Make sure you post it when you are done.  Looks exceptionally smooth with the putty!

**Reply by Looooopy** [reply]
Almost done with paintjob should just paint an eye and small fixes when it has dried a couple of hours
![image](https://user-images.githubusercontent.com/16755871/147111372-0fdc7a2a-07a4-460e-9258-2d95c145c512.png)

**Reply by Looooopy** [reply]
Started to print right side of keyboard now its only 60hours left!

-- See if I have fixed my clogging problem;
    Replaced the old PTFE tube with a new Capricorn tube 
    Replaced the PTFE fitting with metal teeth's 
    Used blowtorch on the nozzle to get it extra clean
    Painted with acrylic white on the new PTFE tube to see if something moves

**Reply by Looooopy** [reply]
And here is the eye in place

![image](https://user-images.githubusercontent.com/16755871/147120759-cb2676dc-3835-4521-9273-4a0e7a580039.png)

**Reply by Looooopy** [reply]
Added wrist rest, the only bad thing is that I need post process that as well so it do look the same

![image](https://user-images.githubusercontent.com/16755871/147789510-c43eef66-3e29-4b62-95ec-ca7a8fcab6ce.png)

**Comment by Looooopy** [comment]
Wiring done :)

May have to use glue gun on some of the hot swaps

![20220107_162252](https://user-images.githubusercontent.com/16755871/148610281-5606b5a2-50d7-4b7c-a036-601468df52b4.jpg)

**Comment by Looooopy** [comment]
Here is almost my wiring done:
![20220108_160749](https://user-images.githubusercontent.com/16755871/148651791-a0dd2908-dcd9-4d57-91d0-85bfe087818f.jpg)

Here is the final left side

https://user-images.githubusercontent.com/16755871/148652651-59ef6247-e8fd-4234-8ab8-245928aa42a2.mp4

**Comment by Looooopy** [comment]
Second case printed after a lot fiddling to resolve my printer issues that I came to the conclusion that I got heat creep because I am using a cabinet so when I printed that log run print it got to hot in the cabinet so the fan for my heat block was not enough but by opening the cabinet doors and print I got my 76 hour long print done :)

Its still not good enough to use without postprocessing but I don't have to do as much post processing like last time and I also knocked my head into the door so the print jumped a millimeter at 1 cm height into the print :) 
I also can see that I need to tighten my x axis screws after so many failed print because I get some bad lines
 
![20220423_135037](https://user-images.githubusercontent.com/16755871/164893568-c4e25ca1-4271-45b9-a633-9991624b4c7d.jpg)
![20220423_135059](https://user-images.githubusercontent.com/16755871/164893590-1113ccea-f7b5-4e6c-9158-8495deca159e.jpg)
![20220423_135113](https://user-images.githubusercontent.com/16755871/164893631-00aa517f-7116-4be2-ae7b-5e7d95e2558b.jpg)

**Reply by Looooopy** [reply]
Put on some putty

![image](https://user-images.githubusercontent.com/16755871/166144812-d9e772b4-befd-4cf5-b2c7-70da47cff164.png)

![image](https://user-images.githubusercontent.com/16755871/166144834-708afa19-23ff-4f65-850b-bf094425f840.png)

![image](https://user-images.githubusercontent.com/16755871/166144855-b6bc6e52-7657-4cea-911c-b622b0119d8e.png)

![image](https://user-images.githubusercontent.com/16755871/166144868-c74cbe21-500b-41db-91e7-439c99a46c60.png)


White paint

![image](https://user-images.githubusercontent.com/16755871/166144905-fbf495a7-feae-4d61-9bdf-ae5a4536aec3.png)

![image](https://user-images.githubusercontent.com/16755871/166144917-e9fe5ac0-9c27-453d-9523-ecf5314ad171.png)

![image](https://user-images.githubusercontent.com/16755871/166144925-2e7ad71a-fc85-479c-af56-1c69c8d308fd.png)

![image](https://user-images.githubusercontent.com/16755871/166144894-ac673d8c-006d-4c20-bbad-65ed6ad93920.png)

**Comment by Looooopy** [comment]
just some varnish and they will look the same ;)
![20220514_030354_2](https://user-images.githubusercontent.com/16755871/168405947-a76d7aac-b6aa-4b00-be13-e46602d1a4ee.jpg)

**Reply by Looooopy** [reply]
Now its soon done I need to reprint the right bottom plate (maybe left as well) but the electronic works.

![image](https://user-images.githubusercontent.com/16755871/170587299-5d59128a-75ce-465d-ab89-5495587a9f7b.png)

I was forced to remove printed hot swap holder on the right side because of I have a bad printing experience with under extrusion. 

I ordered some upgrades to see if I can get it to work better because as it is now its no point having a printer that get clogged and giving me headaches :)

**Reply by joshreve** [reply]
Looks good.  When I've had under extrusion it is normally the Bowden tube backing out of the extruder, creating a cool partial blockage.  I open the top, heat it up and use a crappy small screwdriver or something thin and long to clear it out and file with the small pins to clear the nozzle.  When you rebuild, make sure the tube is cut flush, leave the shark bite fitting a quarter or eighth turn from being tight, push to the bottom and then tighten the fitting.  It really seals the tube to the bottom of the counter bite to minimize the opportunity for it to happen again.

This may not be the problem you are having, but it is the problem I've had most often and is hard to diagnose.

**Reply by Looooopy** [reply]
> Looks good. When I've had under extrusion it is normally the Bowden tube backing out of the extruder, creating a cool partial blockage. I open the top, heat it up and use a crappy small screwdriver or something thin and long to clear it out and file with the small pins to clear the nozzle. When you rebuild, make sure the tube is cut flush, leave the shark bite fitting a quarter or eighth turn from being tight, push to the bottom and then tighten the fitting. It really seals the tube to the bottom of the counter bite to minimize the opportunity for it to happen again.
> 
> This may not be the problem you are having, but it is the problem I've had most often and is hard to diagnose.

Thanks and thanks for the tip I think I have pulled all the parts down multiple times

* Replaced Bowden tube with clean cuts (done multiple times)
* The latest test was using Cheps advise just to have small Bowden tube inside nozzle that cant move
* Replace nozzle with new one
* Tighten nozzle at 250 Celsius
* Tighten nozzle unscrew half a counter then tighten the Bowden connect to get a snug fit with Bowden tube.
* Check heat block fan it seams to work properly (until today when the end of my filament went in it and and a fang got destroyed)
* Print with open doors on my cabinet
* New filament spool
* Humidity is not an issue here either
* Test to increase print temp 5 degrees in multiple stages
* Test to increase flow rate 2,5% in multiple stages

Still every print got under extrusion after 10-20h print  and I have no clue so I think why throw even more time on this maybe I get less pain if I upgrade. I have already wasted 2 weeks fiddling (work time) on it and have not been able to identify why It started to under extrude. 

 Got so angry after the fan got damaged so I ordered a complete new setup
* Micro-swiss NG -> No more Bowden tube inside and direct drive
* New fan :)
* BIGTREETECH TFT35 E3 v3.0 (replace after I have got clean result  with just the Micro-swiss)
* BIGTREETECH SKR mini E3 v3.0 (replace after I have got clean result  with just the Micro-swiss)

So either the upgrade works or I take a break from printing :)

**Reply by Looooopy** [reply]
Replacing my hotend did the trick but I think I know the problem after taking it down again.

The metal tube that goes into the heatsink was lose a quarter of a round almost not noticeable because I had taken it apart to check for clogs multiple times.

I did not test because I was feed up with it, so I just replaced the whole hotend!
But if someone else get the same issue they can try it out.

**Comment by Looooopy** [comment]
Done - if I should not change processor and insert LED:s and a battery :)

![20220608_012023](https://user-images.githubusercontent.com/16755871/172670981-c4658398-30cb-4d3e-ab84-8a2982bcd8cc.jpg)

**Reply by Looooopy** [reply]
it was an heavy colossus 700gram each a total of 1.4KG with wrist support

**Comment by bosd** [comment]
What settings did you sue for the Oled clip? How is that one working for you?

**Reply by Looooopy** [reply]
could not find my settings but I did start from these settings

https://github.com/joshreve/dactyl-keyboard/discussions/19#discussioncomment-1094723

**Comment by Looooopy** [comment]
Got tired of space restrictions of ProMicro so now I installed two STM32F411
Did also change the keycaps and inserted some ws2812b LEDs :)
![20220724_132810](https://user-images.githubusercontent.com/16755871/180645305-b77f6f17-ab58-4360-a7c9-e604f95695dd.jpg)

**Reply by Looooopy** [reply]
After two years of use, becoming dependent on the keyboard and getting tired of the mac keyboard now I am starting building my second keyboard.
It fits better on the hot swap holder without under extrusion, but I have forgot how long things take to do =)
